examples/cactus-check-connection-ethereum-validator/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "check-connection-ethereum-validator",
+  "version": "1.0.0-rc.3",
+  "private": true,
+  "scripts": {
+    "start": "node ../../dist/packages/cactus-cmd-socketio-server/src/main/typescript/routing-interface/www.js",
+    "debug": "nodemon --inspect ../../dist/packages/cactus-cmd-socketio-server/src/main/typescript/routing-interface/www.js",
+    "build": "npm run build-ts && npm run copy-static-assets && npm run copy-blp-config && npm run replace-blp-config-path",
+    "build-ts": "tsc -p ./tsconfig.json",
+    "tslint": "tslint -c tslint.json -p tsconfig.json './*.ts'",
+    "copy-static-assets": "ts-node copyStaticAssets.ts",
+    "copy-blp-config": "ts-node copyBLPConfig.ts",
+    "replace-blp-config-path": "ts-node replaceBLPConfigPath.ts",
+    "init-check-connection-ethereum-validator": "ln -s ../examples/cactus-check-connection-ethereum-validator/node_modules ../../dist/node_modules"
+  },
+  "dependencies": {
+    "@types/node": "^14.0.24",
```

Comment:
@StepniowskiP Would it be okay to lock the versions down to exact ones in the interest of us trying to get closer to [reproducible builds](https://reproducible-builds.org/)?

Reply:
Sure, I can set types/node version so it will match the one in project root.

Reply:
@StepniowskiP Thank you very much!

Reply:
@petermetz I have updated version in latest commit.

Reply:
@StepniowskiP Cool, thanks so much!